### DISCUSSION TITLE
Students can't clone assignments from Classroom (500 error)

### DISCUSSION THREAD

**Discussion by timalcon** [community/community#130805] [GitHub Education] [answered]
### Hi everyone,

Students in courses using GitHub Classroom haven't been able to clone assignments since yesterday. They keep getting a 500 error (as I do when I try it). I've tried it from links that previously worked, and they also give that error, so I don't think the issue's on our end.

**Comment by joshjohanning** [comment]
There was a service disruption yesterday (Jul 1). Are you still having issues? 

https://www.githubstatus.com/incidents/4dq0ww5d0q7j

**Reply by timalcon** [reply]
Yes, we are still having issues.

**Reply by timalcon** [reply, marked as answer]
Fixed - it seems there was a change on GitHub's end that meant we needed to change an organization setting (to allow forking of private repos).
